test/functional/tools/options_from_metadata_file.xml

```diff
@@ -0,0 +1,29 @@
+<tool id="options_from_metadata_file" name="Test for options from dataset metadata file" version="1.0.0" profile="21.01">
+  <command><![CDATA[
+#assert $input.metadata.species_chromosomes
```

Comment:
Out of curiosity: What is the idea of this assertion?

Reply:
Copied from another test tool. It doesn't seem to be harmful, but I will be happy to remove it.